migrating/ossm-migrating-read-me-assembly.adoc

```diff
@@ -0,0 +1,95 @@
+:_content-type: ASSEMBLY
```

Comment:
Per the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata):
```suggestion
:_mod-docs-content-type: ASSEMBLY
```